resources/benchmark.py

```diff
@@ -0,0 +1,193 @@
+import sys
+
+from numpy.lib import delete
+sys.path.append("/Users/lc21/Documents/KCL_BMEIS/ExeTera")   
```

Comment:
Is this file meant to be included? We should remove this line and clean up the comments below if so.